pandas/core/window/rolling.py

```diff
@@ -922,16 +922,19 @@ class Window(_Window):
     * ``blackmanharris``
     * ``nuttall``
     * ``barthann``
-    * ``kaiser`` (needs beta)
-    * ``gaussian`` (needs std)
-    * ``general_gaussian`` (needs power, width)
-    * ``slepian`` (needs width)
-    * ``exponential`` (needs tau), center is set to None.
+    * ``kaiser`` (needs parameter: beta)
+    * ``gaussian`` (needs parameter: std)
+    * ``general_gaussian`` (needs parameters: power, width)
+    * ``slepian`` (needs parameter: width)
+    * ``exponential`` (needs parameter: tau), center is set to None.
 
     If ``win_type=None`` all points are evenly weighted. To learn more about
     different window types see `scipy.signal window functions
     <https://docs.scipy.org/doc/scipy/reference/signal.html#window-functions>`__.
 
+    Certain window types require a scale parameter to be passed. Please see
```

Comment:
Is this supposed to read `scalar parameter` or is there an actual `scale` argument you are saying needs to be passed?

Minor but once clarified this is good to go

Reply:
Yes, as far as I understand, `power` isn't a scale parameter for the general gaussian distribution - `scalar parameter` or even just `parameter` might be better

Reply:
```suggestion
    Certain window types require additional parameters to be passed. Please see
```